srml/executive/src/tests.rs

```diff
@@ -0,0 +1,246 @@
+// Copyright 2017-2019 Parity Technologies (UK) Ltd.
```

Comment:
It's a new file, no?
```suggestion
// Copyright 2019 Parity Technologies (UK) Ltd.
```